docs/6.x/monitoring.md

```diff
@@ -180,60 +99,89 @@ metadata:
   name: smtp
 spec:
   host: smtp.host
-  port: <smtp port> # 465 by default
+  port: <smtp port>
   username: <username>
   password: <password>
----
+```
+
+Create the SMTP configuration:
+
+```bash
+$ gravity resource create smtp.yaml
+```
+
+The second resource is called `alerttarget`. It defines the alerts email recipient:
+
+```yaml
 kind: alerttarget
 version: v2
 metadata:
   name: email-alerts
 spec:
-  email: triage@example.com # Email address of the alert recipient
+   # email address of the alerts recipient
+  email: triage@example.com
 ```
 
-```bsh
-$ gravity resource create -f smtp.yaml
+Create the target:
+
+```bash
+$ gravity resource create -f target.yaml
 ```
 
-To create new alerts, use another resource of type `alert`:
+!!! note:
+    Currently only a single alerts email recipient is supported.
+
+### Configuring Alerts
 
+Defining new alerts is done via a Gravity resource called `alert`:
 
 ```yaml
 kind: alert
 version: v2
 metadata:
-  name: my-formula
+  name: cpu-alert
 spec:
+  # the alert name
+  alert_name: CPUAlert
+  # the rule group the alert belongs to
+  group_name: test-group
+  # the alert expression
   formula: |
-    Kapacitor formula
+    node:cluster_cpu_utilization:ratio * 100 > 80
+  # the alert labels
+  labels:
+    severity: info
+  # the alert annotations
+  annotations:
+    description: |
+      This is a first test alert
```

Comment:
nit: this should ideally be consistent with the example, i.e. `Alert when CPU utilization exceeds 80%`